VueJS/ToDoApp/src/components/ToDoItem.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <b-list-group-item>
+    <span v-if="editModeOn">
+      <input ref="editRef" v-model="todoEditTextInput" value="todo.text" @keydown.enter="save">
```

Comment:
`@keydown.enter="save"` :heart_eyes:

Reply:
```suggestion
      <input ref="editRef" v-model="todoEditTextInput" @keydown.enter="save">
```
Why do you need `value="todo.text"`? I couldn't see the difference when I removed it.